Library/Homebrew/dev-cmd/pr-pull.rb

```diff
@@ -501,6 +504,15 @@ def self.pr_pull
           upload_args << "--root-url-using=#{args.root_url_using}" if args.root_url_using
           safe_system HOMEBREW_BREW_FILE, *upload_args
         end
+      ensure
+        if args.retain_bottle_dir? && ENV["GITHUB_ACTIONS"]
+          ohai "Bottle files retained at:", dir
```

Comment:
```suggestion
          ohai "Bottle files retained at: #{dir}"
```

Nit: style

```rb
brew(main):003:0> dir = "/tmp"
=> "/tmp"
brew(main):004:0> ohai "Bottle files retained at:", dir
==> Bottle files retained at:
/tmp
=> nil
brew(main):005:0> ohai "Bottle files retained at: #{dir}"
==> Bottle files retained at: /tmp
=> nil
```

This makes sure the output stays on one line.

Reply:
@apainintheneck `ohai` will truncate long output and doesn't need the colon if on one line so I think this is actually better as-is.